.github/workflows/shared-branch-push-library-java.yml

```diff
@@ -0,0 +1,16 @@
+name: Java Library Branch Push
+
+on:
+  workflow_call:
+
+env:
+  jiraServer: cartoncloud.atlassian.net
+
+jobs:
+  build:
+    name: Build
+    uses: cartoncloud/actions/build-library-java@v3
+    with:
+      pomVersion: ${{ github.sha }}
+      mavenS3AccessKeyId: ${{ secrets.TEMP_MAVEN_S3_ACCESS_KEY_ID }}
+      mavenS3SecretAccessKey: ${{ secrets.TEMP_MAVEN_S3_SECRET_ACCESS_KEY }}
```

Comment:
Do we really need an intermediate shared workflow that calls the build action? I probably worded it poorly in slack but couldn't we just call the action directly from the library workflow file? So just have the above `jobs` element in the services directly?

Reply:
I tried to invoke the shared build action but seems it doesn't work. https://github.com/cartoncloud/common-base/pull/4/files
`[Invalid workflow file: .github/workflows/branch-push.yml#L19](https://github.com/cartoncloud/common-base/actions/runs/3224580670/workflow)
invalid value workflow reference: references to workflows must be rooted in '.github/workflows'`
Assume the error message means that the workflow has to be inside .github/workflows no matter it's shared or internal